containers/wordpress/Dockerfile.template

```diff
@@ -3,6 +3,9 @@ FROM wordpress:latest
 RUN pecl install xdebug && docker-php-ext-enable xdebug
 RUN docker-php-ext-install pdo_mysql
 
+RUN apt-get update
```

Comment:
could you make it a on liner, 
apt-get update && apt-get -y install defualt-mysql-client 
this way only one file layer will bee added not 2